.github/ISSUE_TEMPLATE/moderator-offboarding.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Moderator offboarding"
+about: "for moderators who wish to leave"
+labels: "moderator offboarding"
+assignees: r-multiverse/administrators
+---
+
+I would like to leave the R-multiverse [moderator](https://r-multiverse.org/team.html#moderators) team.
+
+## Offboarding process
+
```

Comment:
the first item should be to contact administrators?

Reply:
and the checklist below should instead be phrased to administrators so they could copy paste it and follow it?

So perhaps even formatted as a code chunk so it might get a copy-paste button?

Reply:
> the first item should be to contact administrators?

I had assumed the new issue itself would (transparently) notify the admins. c2eb62fbe8044e23ba4aa7f8a0096c8f1bdd7853 adds explicit mentions, and 4f4e9b88aa8326d65a6ac961296dcd24319907a5 makes this clearer. Is there anything specific we should do in addition to this?

Reply:
> and the checklist below should instead be phrased to administrators so they could copy paste it and follow it?

Would it be more convenient to use the clickable checkboxes in the issue itself? This feels like what [task lists](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists) are designed for.

Reply:
Oh right